tests/Console/Report/FixReport/GitlabReporterTest.php

```diff
@@ -29,6 +30,11 @@ final class GitlabReporterTest extends AbstractReporterTestCase
 {
     use AssertJsonSchemaTrait;
 
+    public static function setUpBeforeClass(): void
+    {
+        parent::setUpBeforeClass();
+    }
+
```

Comment:
Is it actually needed, since it does not do anything more than parent?

Reply:
old left over, thanks